script.js

```diff
@@ -0,0 +1,306 @@
+const suits = [
+    {
+        suit: '♣',
+        color: 'black'
+    },
+    {
+        suit: '♠',
+        color: 'black'
+    },
+    {
+        suit: '♥',
+        color: 'red'
+    },
+    {
+        suit: '♦',
+        color: 'red'
+    }
+]
+
+const faces = [
+    {
+        face: 'A',
+        value: 1
+    },
+    {
+        face: '2',
+        value: 2
+    },
+    {
+        face: '3',
+        value: 3
+    },
+    {
+        face: '4',
+        value: 4
+    },
+    {
+        face: '5',
+        value: 5
+    },
+    {
+        face: '6',
+        value: 6
+    },
+    {
+        face: '7',
+        value: 7
+    },
+    {
+        face: '8',
+        value: 8
+    },
+    {
+        face: '9',
+        value: 9
+    },
+    {
+        face: '10',
+        value: 10
+    },
+    {
+        face: 'J',
+        value: 11
+    },
+    {
+        face: 'Q',
+        value: 12
+    },
+    {
+        face: 'K',
+        value: 13
+    }
+]
+
+const slider = document.getElementById("difficulty")
+const startBtn = document.getElementById("start")
+let side = 'R';
+let difficulty = 0
+let cards = [];
+let drawnCards = [];
```

Comment:
Do you need to keep an array of all of the drawn cards? Aren't we just comparing 2?